package.json

```diff
@@ -58,6 +58,7 @@
     "homepage": "https://github.com/zowe/api-layer#readme",
     "dependencies": {},
     "devDependencies": {
-        "concurrently": "6.0.0"
+        "concurrently": "6.0.0",
+        "yaml": "^1.10.2"
```

Comment:
Please remove the yaml dependency from the main package.json, since we need it included only in the api-catalog-ui one